app/presenters/flow_presenter.rb

```diff
@@ -42,7 +42,7 @@ def start_node
     @start_node ||= @flow.start_node.presenter(flow_presenter: self)
   end
 
-  def change_answer_link(question)
+  def change_answer_link(question, forwarding_responses)
```

Comment:
```rb
def change_answer_link(question, forwarding_responses = {} )
```
non-blocker: I guess we could've made the second argument optional by setting an empty object as default.